src/pages/workspace/WorkspaceInviteMessagePage.js

```diff
@@ -191,6 +191,7 @@ class WorkspaceInviteMessagePage extends React.Component {
                                 size={CONST.AVATAR_SIZE.LARGE}
                                 icons={OptionsListUtils.getAvatarsForAccountIDs(_.values(this.props.invitedEmailsToAccountIDsDraft), this.props.personalDetails)}
                                 shouldStackHorizontally
+                                shouldStackVertically
```

Comment:
Having both `shouldStackHorizontally` and `shouldStackVertically` sound confusing. Can we rename them better based on their use?